src/contracts/interfaces/GPv2ERC1271.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.7.6;
+
+library ERC1271 {
+    /// @dev Value returned by a call to `isValidSignature` if the signature
+    /// was verified successfully. The value is defined in the EIP-1271 standard
+    /// as: bytes4(keccak256("isValidSignature(bytes32,bytes)")
+    bytes4 internal constant MAGICVALUE = 0x1626ba7e;
+}
+
+/// @title ERC1271 Interface
+/// @dev Standardized interface for an implementation of smart contract
+/// signatures as described in the EIP-1271 standard. The code that follows is
+/// identical to the code in the standard with the exception of formatting and
+/// syntax changes to adapt the code to our Solidity version.
+abstract contract ERC1271Verifier {
```

Comment:
nit: I think this should probably be an interface instead of an abstract contract:
```suggestion
interface IERC1271Verifier {
```